source/routing/raptor_api.cpp

```diff
@@ -187,20 +187,59 @@ static void co2_emission_aggregator(pbnavitia::Journey* pb_journey){
 
 static void compute_durations(pbnavitia::Journey* pb_journey) {
     uint32_t total_walking_duration = 0;
+    uint32_t total_car_duration = 0;
+    uint32_t total_bike_duration = 0;
     for (const auto& section: pb_journey->sections()) {
         if ((section.type() == pbnavitia::STREET_NETWORK && section.street_network().mode() == pbnavitia::StreetNetworkMode::Walking) ||
         (section.type() == pbnavitia::TRANSFER && section.transfer_type() == pbnavitia::walking) ||
         (section.type() == pbnavitia::CROW_FLY && section.street_network().mode() == pbnavitia::StreetNetworkMode::Walking)) {
             total_walking_duration += section.duration();
         }
+
+        if (section.type() == pbnavitia::STREET_NETWORK) {
```

Comment:
CROW_FLY must be handled. Maybe TRANSFER if it can be mutualised, even if we never do something else than walking in transfer for the moment. It should be possible to mutualise with the test for walking.